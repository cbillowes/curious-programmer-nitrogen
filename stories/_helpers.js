import React from 'react'
import Layout from '../src/components/layout'
import { SmallSpacer } from '../src/components/spacer'

const Page = ({ className, children }) => {
  return (
    <Layout
      className={className}
    >
      <SmallSpacer />
      {children}
    </Layout>
  )
}

export default Page