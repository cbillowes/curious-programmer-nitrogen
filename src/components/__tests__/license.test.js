import React from 'react'
import License from '../license'
import { getSnapshot } from './_helpers'

describe(`License`, () => {
  it(`should render`, () => {
    const license = (
      <License />
    )
    const tree = getSnapshot(license)
    expect(tree).toMatchSnapshot()
  })
})
