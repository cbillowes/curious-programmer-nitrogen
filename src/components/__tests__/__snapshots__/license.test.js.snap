// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`License should render 1`] = `
<div
  className="license"
>
  <p>
    <span
      className="creative-commons-logo"
    >
       
      <a
        className="plain"
        href="http://creativecommons.org/licenses/by-sa/4.0/"
        rel="nofollow noopener noreferrer"
        target="_blank"
      >
        <img
          alt="Creative Commons License"
          src="https://i.creativecommons.org/l/by-sa/4.0/88x31.png"
          title="Creative Commons Attribution-ShareAlike 4.0 International License"
        />
      </a>
       
    </span>
    The content is written by 
     
    <a
      className="underline-only"
      href="/about"
    >
      Clarice Bouwer
    </a>
     
     and is licensed under the
     
     
    <a
      className="underline-only"
      href="https://creativecommons.org/licenses/by-sa/4.0/"
      rel="nofollow noopener noreferrer"
      target="_blank"
    >
      Creative Commons Attribution-ShareAlike 4.0 International License
    </a>
     
    . References to external resources are subject to their own terms and conditions. There is a 
     
    <a
      className="underline-only"
      href="/license"
    >
      license
    </a>
     
     
    for the code. It's probably a good idea to read the 
     
    <a
      className="underline-only"
      href="https://www.iubenda.com/privacy-policy/7895922"
      rel="nofollow noopener noreferrer"
      target="_blank"
    >
      privacy policy
    </a>
     
    . The website is open-source and is available on 
     
    <a
      className="underline-only"
      href="https://github.com/cbillowes/curious-programmer-nitrogen"
      rel="nofollow noopener noreferrer"
      target="_blank"
    >
      GitHub
    </a>
     
    . Please give it a star if you think it is cool.
    <br />
    Icons made by
     
    <a
      href="https://www.flaticon.com/authors/freepik"
      title="Freepik"
    >
      Freepik
    </a>
     
    from
    <a
      href="https://www.flaticon.com/"
      title="Flaticon"
    >
      www.flaticon.com
    </a>
  </p>
</div>
`;
